core-dev/packages/coqide/coqide.8.13+beta1/opam

```diff
@@ -0,0 +1,51 @@
+opam-version: "2.0"
+maintainer: "coqdev@inria.fr"
+authors: "The Coq development team, INRIA, CNRS, and contributors."
+homepage: "https://coq.inria.fr/"
+bug-reports: "https://github.com/coq/coq/issues"
+dev-repo: "git+https://github.com/coq/coq.git"
+license: "LGPL-2.1-only"
+synopsis: "IDE of the Coq formal proof management system"
+description: """
+CoqIDE is a graphical user interface for interactive development
+of mathematical definitions, executable algorithms, and proofs of theorems
+using the Coq proof assistant.
+"""
+
+depends: [
+  "coq" {= version}
+  "ocamlfind" {build}
+  "conf-findutils" {build}
+  "lablgtk3-sourceview3"
+  "conf-adwaita-icon-theme"
+]
+build: [
+  [
+    "./configure"
+    "-configdir" "%{lib}%/coq/config"
+    "-prefix" prefix
+    "-mandir" man
+    "-docdir" doc
+    "-libdir" "%{lib}%/coq"
+    "-datadir" "%{share}%/coq"
+    "-native-compiler" "yes" {coq-native:installed} "no" {!coq-native:installed}
```

Comment:
@gares @proux01 @Zimmi48 @palmskog, sorry to ping you… do you think the `-native-compiler` option is needed for the coqide package?

Reply:
I did set it to keep the invariant that the configure is called with the same options. I believe it is not important.

Reply:
My rationale here is that:
- if it is irrelevant, it won't hurt
- if CoqIDE starts to understand something about native, then it is better to be configured this way